### ISSUE TITLE
"You have enabled the module token-health, but it's dependencies lib-df-hotkeys are not enabled!"

### ISSUE THREAD

**Issue by ronaldsf1977** [schlosrat/token-health#39]
Foundry v0.7.9
PF2e v1.15.0.5631

I am getting the error message in the title when I try to active the Token Health module. When the dialog box saying that I need to activate a module it depends on pops up, I say activate it. Still, the message is coming up and is preventing me from closing the Manage Modules window

**Comment by schlosrat** [comment]
Thanks for reporting this.

The only dependency is Library: DF Hotkeys. Do you have that module installed?

**Comment by ronaldsf1977** [comment]
Ah, I do not.
Installed it now, and I can activate Token Health now.
This error has never come up before. I have been using Token Health for months, and I've never had Library: DF Hotkeys installed. Is this a new dependency?
(I had "Layer Hotkeys" installed and activated for the past few months and I deactivated it. Perhaps that triggered the problem?)

**Comment by schlosrat** [comment]
Yep, this is a brand new dependency in Token Health v 0.4.0. It wasn't a part of any prior version.

I had to scrap the old hot key system as it's not compatible with 0.8.x. DF Hotkeys was the one recommended by some other developers so I went with that. I did list it as a dependency in the module.json file, so Foundry ought to automatically download it when you install Token Health. It's possible that works differently if you're upgrading from a previous versions than if you're installing from scratch. This sounds to me more like it might be an issue with how Foundry handles the dependency in an update, so there may not be much else I can do.

On the plus side you're all set for using it in 0.8.x!
